### PR TITLE
Add swagger test

### PR DESCRIPTION
**Pull request by Uxio0**
- Detects issues with swagger breaking
- Related to #1531

### CODE REVIEW

File: safe_transaction_service/history/tests/test_views.py [hunk 1]
```diff
@@ -69,6 +69,15 @@ def test_about_view(self):
...
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_swagger(self):
+        url = reverse("schema-json", args=(".json",))
+        response = self.client.get(url, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        url = reverse("schema-swagger-ui")
+        response = self.client.get(url, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```
Comment (fmrsabino) [inline safe_transaction_service/history/tests/test_views.py:79]: Out of curiosity: was the swagger interface [here](https://github.com/safe-global/safe-transaction-service/pull/1526) returning an error code or was it returning `200` while rendering an error message? 🤔

**Review by fmrsabino — APPROVED** [review]
🚀

File: safe_transaction_service/history/tests/test_views.py (hunk 1 above)
Comment (Uxio0) [reply safe_transaction_service/history/tests/test_views.py:79]: It was returning `200`, indeed. Error was on client side when requesting the `json` schema

File: safe_transaction_service/history/tests/test_views.py [hunk 2]
```diff
@@ -69,6 +69,15 @@ def test_about_view(self):
...
+
```
[the remark this answers is not in the document]
Comment (Uxio0) [reply safe_transaction_service/history/tests/test_views.py:80]: You are right, done

**Pull request merged**